.circleci/config.yml

```diff
@@ -2,10 +2,10 @@ version: 2.1
 
 # reusable environment for all jobs
 executors:
-  node-erbium-executor:
+  node-fermium-executor:
```

Comment:
Could we just name the executor `node-executor` so we don't have to update it in multiple places for every version? Just have the executor running the proper image, like the change on line 8.

Reply:
good point, will do 👍🏼